lua/gatekeeperc.lua

```diff
@@ -24,12 +24,21 @@ struct gk_config {
 	/* This struct has hidden fields. */
 };
 
+struct ggu_config {
+	unsigned int      lcore_id;
+	uint16_t          ggu_src_port;
+	uint16_t          ggu_dst_port;
+	/* This struct has hidden fields. */
+};
+
 ]]
 
 -- Functions and wrappers
 -- TODO Define the C functions for other functional blocks.
 ffi.cdef[[
 
+void rte_free(void *ptr);
```

Comment:
`rte_free()` is being exported here only to release `ggu_conf`, but `ggu_conf` already has `cleanup_ggu()` to do the job.